### DISCUSSION TITLE
ye

### DISCUSSION THREAD

**Discussion by Outmoddedsoup436** [remarkablegames/inversion#477] [General]
Question 1 (1 point) The main purpose of the planer is to: 
a. plane the edge of the stock straight and square to one face. 
b. plane the faces of the stock flat, parallel to each other, and to the desired thickness. 
c. plane the ends of the stock 90 degrees to the edges. 
d. plane a smooth, graceful curve on the edge of the stock. 

Question 2 (1 point) The main purpose of the jointer is to: 
a. plane the edge of the stock straight and square to the face. 
b. plane the faces of the stock flat, parallel to each other, and to the desired thickness. 
c. plane the ends of the stock 90 degrees to the edges. 
d. plane a smooth, graceful curve on the edge of the stock. 

Question 3 (1 point) The shortest piece of wood that you can safely run through the planer or jointer is: 
a. 4 inches
 b. 8 inches 
c. 12 inches 
d. 16 inches 

Question 4 (1 point) The maximum depth of cut you may make, in this class, on the planer or jointer is: 
a. 1/32” 
b. 1/16” 
c. 1/8” 
d. 1/4” 

Question 5 (1 point) Before running wood through the Planer or Jointer, check to be sure it is free of: 
a. loose knots
 b. nails 
c. dirt 
d. all of the above e none of the above 

Question 6 (1 point) For your first pass on the Planer, the depth of cut should be set to: 
a. 1/16"
 b .1/8" 
c. the desired thickness of the wood. 
d. the present thickness of the wood. 

Question 7 (1 point) To get the smoothest cut on the Surface Planer, the direction of the grain on the edge of the wood should run: a in a wavy pattern, up and down b downwards to the rear c upwards to the rear d none of the above are correct Question 8 (1 point) To get the smoothest cut on the Jointer, the direction of the grain on the edge of the wood should run: a in a wavy pattern, up and down b downwards to the rear c upwards to the rear d none of the above are correct Question 9 (1 point) When jointing stock that is taller than the height of the fence, you must always use a: a back-up board b stop-block c top push stick d side push stick e both C and D are correct Question 10 (1 point) When jointing stock that is shorter than the height of the fence, you must always use a: a back-up board b stop-block c top push stick d side push stick e both C and D are correct Question 11 (1 point) When using the Planer or Jointer, run your wood through so that it cuts: a with the grain b across the grain c perpendicular to the grain d at a 90 degree angle to the grain Question 12 (1 point) If stock becomes stuck in the Planer or Jointer you should: a reach in and pull it out. b ask a helper to reach in and pull it out. c keep you hands away from the blades and shut it off d scream Question 13 (1 point) When surfacing many boards of various thicknesses, you should plane the thickest boards first. True False Question 14 (1 point) When using the jointer what should you joint first? a an edge b a face c it doesn't matter Question 15 (1 point) Where should you stand when using the jointer a in the middle by the cutterhead. b at the front of the infeed table, feet fixed on the ground. c By the side of the infeed table, slowly moving with the work piece. d At the end of the outfeed table. Question 16 (1 point) It's fine to rest your hand on the cutterhead guard. True False Question 17 (1 point) If the wood is large enough you don't need push blocks. True False Question 18 (1 point) When jointing a face, how do use the push blocks? a Push them down as hard as possible on the infeed side. b use them to let the wood glide and skip over the cutterhead. c slightly firm pressure to keep the board stable on both the infeed and outfeed table. d push blocks are not necessary. Question 19 (1 point) To joint an edge, always put the jointed face against the fence, so the two will be perpindicular to one another. True False Question 20 (1 point) It's ok to put your hands under the cutterhead in the planer. True False

**Comment by remarkablemark** [comment]
Good to know
